apps/trading/components/orders-container/orders-container.tsx

```diff
@@ -40,7 +38,7 @@ export const OrdersContainer = ({ marketId, filter }: OrderContainerProps) => {
   });
 
   if (!pubKey) {
-    return <Splash>{t('Please connect Vega wallet')}</Splash>;
```

Comment:
I think we still want this text. Rather than just rendering a blank pane. Same for the other container components

Reply:
done